dds2val/src/idls/vss/setup.py

```diff
@@ -0,0 +1,4 @@
+import setuptools
```

Comment:
Do we need a license header also for very small python files, well possibly even for empty ones like `__init__.py`? (FYI: @SebastianSchildt )

Reply:
These are no files which include intelectual property. So I assume they need no license headers like config files and so on.